pkgs/servers/sql/postgresql/ext/pgddl/default.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  fetchFromGitHub,
+  postgresql,
+  buildPostgresqlExtension,
+  postgresqlTestExtension,
+  perl,
+  replaceVars,
+}:
+buildPostgresqlExtension (finalAttrs: {
+  pname = "pgddl";
+  version = "0.29";
+
+  src = fetchFromGitHub {
+    owner = "lacanoid";
+    repo = "pgddl";
+    tag = finalAttrs.version;
+    hash = "sha256-W3G6TGtkj+zXXdGZZR0bmZhsLuFJvuGTlDoo8kL8sf0=";
+  };
+
+  buildInputs = [ perl ];
+
+  patches = [
+    (replaceVars ./0001-replace-shebangs.diff {
+      perl = lib.getExe' perl "perl";
```

Comment:
Why not just `lib.getExe perl`?

Reply:
Just personal preference. Feels more explicit. Happy to use the single arg version in future if that's preferred. For now I'm going to get rid of this and try the patchShebang func.